packages/generators/init-template/webpack-template/prettier.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  singleQuote: true,
+  trailingComma: 'es5',
+  arrowParens: 'always',
+};
```

Comment:
Let's remove it in favor of prettier defaults.

Reply:
Yes, let's do it

Reply:
done